### PR TITLE
Update postorius to reflect Systers

### LINKED ISSUE #23: Update Postorius to reflect changes from main developement repo
## Description
Postorius has changed a lot since it was last cloned in 2015, updates need to be made to reflect updates at [https://gitlab.com/mailman/postorius](https://gitlab.com/mailman/postorius)

## Acceptance Criteria

- [x] Make require update

## Definition of Done
- [x] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation
4 hours

### PR DESCRIPTION
**Pull request by Monal5031**
### Description
This PR updates the repo to reflect Systers' and also making it easy for setup via mentioning the required steps in README itself.

Fixes #23 

### Type of Change:

- Quality Assurance
- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
N/A

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by Monal5031** [comment]
ping @MehaKaushik @naman1901 @ayushgarg1804 @pavankm Please review.

**Comment by Monal5031** [comment]
@naman1901 can you view the rich diff in files changed?

**Comment by naman1901** [comment]
I think it is an issue at my end, because the syntax seems right, but the hyperlinks and formatting aren't building. I will approve if you can confirm that the formatting is okay at your end.

xyz

File: README.rst [hunk 1]
```diff
@@ -1,75 +1,83 @@
...
+**Please note**, the `live documentation <http://postorius.readthedocs.io/en/latest/development.html>`_ contains many other details that are important but may not be required for a successful setup, but it is strongly encouraged to visit the live documentation.
 
-Requirements
-============
 
-Postorius requires Python 2.7 or newer and mailmanclient,
-the official Python bindings for GNU Mailman.
-The minimum Django version is 1.8.
-Postorius needs a running version of GNU Mailman version 3.
+If you face any issues while setting up Postorius locally, ask them on slack on `#mailman3 <https://systers-opensource.slack.com/messages/C0QK5PCNS/>`_ channel.
```
Comment (naman1901) [inline README.rst]: ask who?

Rephrase: post your queries on the mailman3 slack channel

File: README.rst [hunk 2]
```diff
@@ -1,75 +1,83 @@
...
 
-Postorius is distributed in the hope that it will be useful,
-but WITHOUT ANY WARRANTY; without even the implied warranty of
-MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser
-General Public License for more details.
 
-You should have received a copy of the GNU Lesser General Public License
-along with mailman.client. If not, see <http://www.gnu.org/licenses/>.
 
+**Please note**, the `live documentation <http://postorius.readthedocs.io/en/latest/development.html>`_ contains many other details that are important but may not be required for a successful setup, but it is strongly encouraged to visit the live documentation.
```
Comment (naman1901) [inline README.rst]: ...but you are...

**Pull request merged**